core-services/prow/02_config/_plugins.yaml

```diff
@@ -719,6 +719,31 @@ plugins:
   - wip
   - yuks
 
+  coreos/rpm-ostree:
```

Comment:
If you envision turning these on for more repos in the future, we can do it at the org level as well

Reply:
I need to have a larger discussion about that...there's a *lot* of repos in coreos/ 
But good to know!